bitnami/mongodb/values.yaml

```diff
@@ -726,6 +726,10 @@ service:
   ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
   ##
   loadBalancerSourceRanges: []
+  ## @param service.allocateLoadBalancerNodePorts Wheter to allocate node ports when service type is LoadBalancer
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#load-balancer-nodeport-allocation
+  ##
+  allocateLoadBalancerNodePorts: true 
```

Comment:
My bad, Didn't test this case.
Currently figuring out how to default this value to the origin default without leaving it out completely..

Reply:
the logic is now fixed. Missed the part where I check for the service type in the svc.yaml
Also, the values are now handled as real booleans and not as strings, hence the appearance in the values.yaml is required.
I guess your points should be resolved with this commits.

Speaking of commits.. 
Since the checks forced me to commit several times, should I stash my changes together to a single commit?